src/blob_format.cc

```diff
@@ -133,18 +133,23 @@ bool operator==(const BlobIndex& lhs, const BlobIndex& rhs) {
 void BlobFileMeta::EncodeTo(std::string* dst) const {
   PutVarint64(dst, file_number_);
   PutVarint64(dst, file_size_);
+  PutVarint64(dst, file_entries_);
+  PutVarint32(dst, file_level_);
 }
 
 Status BlobFileMeta::DecodeFrom(Slice* src) {
-  if (!GetVarint64(src, &file_number_) || !GetVarint64(src, &file_size_)) {
+  if (!GetVarint64(src, &file_number_) || !GetVarint64(src, &file_size_) ||
+      !GetVarint64(src, &file_entries_) || !GetVarint32(src, &file_level_)) {
```

Comment:
you should consider the compatibility issue

Reply:
We should add new metadata to `kAddedBlobFileV2`. See https://github.com/pingcap/titan/pull/51. Let's reuse the same  tag `kAddedBlobFileV2` for both of the changes.